realm/src/androidTest/java/io/realm/internal/JNITableTest.java

```diff
@@ -49,14 +49,6 @@ public void testTableToString() {
         assertEquals(expected, t.toString());
     }
 
-    public void testGroupEquals() {
```

Comment:
Because the equals method was removed and was substituted with Table.hasSameSchema() I will add unit tests for the new method instead, which has different semantics from equals. This particular test for example would fail because the schema would be the same even after adding an empty row.